### PR TITLE
warnings, 80 col and indent

### PR DESCRIPTION
**Pull request by mithunshashidhara**

### CODE REVIEW

**Comment by mithunshashidhara** [comment]
this commit fixes warnings in poet2 code, corrects comments as per doxygen standards, corrects indentation. fixes 80 char width in code.

**Comment by askmish** [comment]
@mithunshashidhara Please rename the title of the PR appropriately inline with sawtooth-core repo rules.

File: src/core/src/consensus_state.rs [hunk 1]
```diff
@@ -120,86 +115,94 @@ struct Entry{
...
                                        };
        let peerid_vec = Vec::from(validator_info.clone().id);
        let peerid_str = String::from_utf8(peerid_vec).expect("Found Invalid UTF-8");
-       self.validators.insert(peerid_str, ValidatorState{ key_block_claim_count: key_block_claim_count, poet_public_key: validator_info.clone().poet_public_key, total_block_claim_count: total_block_claim_count});
-
-   }
+       self.validators.insert(peerid_str,
+        ValidatorState{ key_block_claim_count: key_block_claim_count,
+                              poet_public_key: validator_info.clone().poet_public_key,
+                      total_block_claim_count: total_block_claim_count});
```
Comment (askmish) [inline src/core/src/consensus_state.rs]: Please fix indentation.

File: src/core/src/consensus_state.rs [hunk 2]
```diff
@@ -120,86 +115,94 @@ struct Entry{
...
-     let peerid_str = String::from_utf8(peerid_vec).expect("Found Invalid UTF-8");
-     let mut validator_state = self.validators.get(&peerid_str);
-     let mut val_state = ValidatorState{ key_block_claim_count: 0, poet_public_key: validator_info.clone().poet_public_key, total_block_claim_count: 0};
-     if validator_state.is_none(){
-      return Box::new(val_state); 
-     }
-     Box::new(validator_state.unwrap().clone())
-   }
+    pub fn get_validator_state(&mut self,  validator_info: ValidatorInfo) ->
+                                                           Box<ValidatorState> {
```
Comment (askmish) [inline src/core/src/consensus_state.rs]: please fix indentation

File: src/core/src/consensus_state.rs [hunk 3]
```diff
@@ -120,86 +115,94 @@ struct Entry{
...
+      let peerid_vec = Vec::from(validator_info.clone().id);
+      let peerid_str = String::from_utf8(peerid_vec).expect("Found Invalid UTF-8");
+      let validator_state = self.validators.get(&peerid_str);
+      let val_state = ValidatorState{ key_block_claim_count: 0,
+                    poet_public_key: validator_info.clone().poet_public_key,
+                    total_block_claim_count: 0};
+      if validator_state.is_none() {
+        return Box::new(val_state);
```
Comment (askmish) [inline src/core/src/consensus_state.rs]: Please fix indentation...

File: src/core/src/consensus_state_store.rs [hunk 4]
```diff
@@ -57,11 +56,11 @@ impl<'a> ConsensusStateStore<'a> {
     pub fn delete(&mut self, block_id: BlockId) -> Result<(), DatabaseError>{
         let mut writer = self.consensus_state_db.writer()?;
         writer.delete(&Vec::from(block_id))?;
-
         Ok(())
     }
 
-    pub fn put(&mut self, block_id: BlockId, consensus_state: ConsensusState) -> Result<(), DatabaseError>{
+    pub fn put(&mut self, block_id: BlockId, consensus_state: ConsensusState) ->
+            Result<(), DatabaseError>{
```
Comment (askmish) [inline src/core/src/consensus_state_store.rs]: Space missing between > and {

File: src/core/src/enclave_sgx.rs [hunk 5]
```diff
@@ -156,7 +157,8 @@ impl EnclaveConfig {
         info!("wait certificate generated is {:?}", wait_cert);
 
         //release wait certificate
-        let status = ffi::release_wait_certificate(&mut eid, &mut wait_cert_info);
+        ffi::release_wait_certificate(&mut eid, &mut wait_cert_info)
```
Comment (askmish) [inline src/core/src/enclave_sgx.rs]: Indentation with respect to the last return line

File: src/core/src/enclave_sim.rs [hunk 6]
```diff
@@ -141,8 +141,7 @@ pub fn initialize_wait_certificate(
     if !in_serialized_prev_block_wait_certificate.is_empty() {
         let deserialized_prev_block_wait_certificate =
                serde_json::from_str(&in_serialized_prev_block_wait_certificate);
-        let mut prev_wait_certificate_obj : WaitCertificate =
-               WaitCertificate::default();
+        let prev_wait_certificate_obj : WaitCertificate;
```
Comment (askmish) [inline src/core/src/enclave_sim.rs]: Please revert to the original line. This one will error out, as we're modifying the var in next few lines.

File: src/core/src/engine.rs [hunk 7]
```diff
@@ -22,24 +22,25 @@ extern crate log4rs;
 use sawtooth_sdk::consensus::{engine::*, service::Service};
 use service::Poet2Service;
 use std::sync::mpsc::{Receiver, RecvTimeoutError};
```
Comment (askmish) [inline src/core/src/engine.rs]: why are the file perms changed to 755

**Pull request closed without merging**